Formula/mage.rb

```diff
@@ -0,0 +1,38 @@
+class Mage < Formula
+  desc "Make/rake-like build tool using Go"
+  homepage "https://magefile.org/"
+  url "https://github.com/magefile/mage.git",
+      :tag => "1.3.0",
+      :revision => "bd5486e7f905b69c168b13088e12fa84d779799e"
```

Comment:
git url is used, because it gets it's version (and commit) through `git` in their `magefile.go` during build. Current version out looks like this:

```
$ mage --version
Mage Build Tool 1.3.0
Build Date: 2018-09-10T20:15:11+02:00
Commit: bd5486e
built with: go1.11
```